acceptance/tests/base/host/host_test.rb

```diff
@@ -316,3 +316,17 @@
     on default, 'true'
   end
 end
+
+step 'Ensure that a long 128+ character string with UTF-8 characters does not break net-ssh' do
```

Comment:
@Iristyle this test passes with `net-ssh 4.0.0` and `net-ssh 3.3.0.beta1`, although you'll have to take my word for with the beta1 version that I've tested manually. With `net-ssh 3.2.0`, it fails with the following error:

 ````
e9r2bzses1bvojd.delivery.puppetlabs.net (windows2012r2-64-1) 13:00:03$ mkdir /tmp/testingareallylongcharacterlengththatisatleast128characterslongnothinglessmoreisfinebutthislineshouldbethatlongwithsomeUTF8attheendAۿᚠ𠜎
    Encoding::CompatibilityError: incompatible character encodings: ASCII-8BIT and UTF-8
    /Users/toneloc/repos/beaker/vendor/bundle/ruby/2.3.0/gems/net-ssh-3.2.0/lib/net/ssh/buffer.rb:284
````